src/common/DOMWorld.ts

```diff
@@ -33,7 +33,7 @@ import { isNode } from '../environment';
 
 // This predicateQueryHandler is declared here so that TypeScript knows about it
 // when it is used in the predicate function below.
-declare const predicateQueryHandler: QueryHandler;
+declare const predicateQueryHandler: Function;
```

Comment:
We seem to have a little loss here by switching to `Function`. Can we define the signature of the query handler like we did before?

Reply:
Ah, yes. We should still be able to something like this here:
```
declare const predicateQueryHandler: (
  element: Element | Document,
  selector: string
) => Element | Element[] | NodeListOf<Element>;
```